server/service/service_hosts.go

```diff
@@ -101,7 +101,7 @@ func (svc Service) GetHostSummary(ctx context.Context) (*fleet.HostSummary, erro
 }
 
 func (svc Service) DeleteHost(ctx context.Context, id uint) error {
-	if err := svc.authz.Authorize(ctx, &fleet.Host{}, fleet.ActionWrite); err != nil {
+	if err := svc.authz.Authorize(ctx, &fleet.Host{}, fleet.ActionList); err != nil {
```

Comment:
Was that deliberate? Seems surprising, but again, not so familiar with how we do policy enforcement.

Reply:
Yes, the idea is that you check that the user can list hosts. Then you get the host, and then you check that the user can write to that host.